packages/nimble-components/src/anchor/styles.ts

```diff
@@ -2,102 +2,88 @@ import { css } from '@microsoft/fast-element';
 import { display } from '@microsoft/fast-foundation';
 import { focusVisible } from '../utilities/style/focus';
 import {
-    linkActiveDisabledFontColor,
     linkActiveFontColor,
     linkActiveProminentFontColor,
     linkDisabledFontColor,
     linkFont,
     linkFontColor,
-    linkProminentDisabledFontColor,
     linkProminentFontColor
 } from '../theme-provider/design-tokens';
 
 export const styles = css`
-    ${display('inline')}
+    @layer base, hover, focusVisible, active, disabled;
```

Comment:
talked offline, two things to doublecheck:
- Are @layer definitions scoped to shadow root? Concerns are other `@` definitions like `@font-face` aren't
- Do we need to align styles with other text-based links that are not sharing the styles directly. i.e. rich-text-editor anchor styling and breadcrumb

Reply:
> Are @layer definitions scoped to shadow root? Concerns are other `@` definitions like `@font-face` aren't

![image](https://github.com/ni/nimble/assets/7282195/7c5b99a2-0136-48f5-a7c2-736e19776bd6)

> Do we need to align styles with other text-based links that are not sharing the styles directly. i.e. rich-text-editor anchor styling and breadcrumb

- The breadcrumb item needed a change to hide the underline when clicking.
- The rich-text editor does not have interactive links and does not use `underline-hidden`, so no changes apply.
- The rich-text viewer had intentionally set the link color to be constant, regardless of interactions, but this seems wrong. The stated intent was "To ensure a consistent appearance, the font color is forced to the default link color regardless of the `href` attribute's presence." An inoperable/disabled link (i.e. no `href`) should render like other disabled links. By removing this bit of styling, we also get the standard interaction colors.

Reply:
I think what is currently there is intentional (and incomplete). The intention is that things that are not a supported link format are treated as plaintext. 

i.e. visually look like plaintext:
![image](https://github.com/ni/nimble/assets/1588923/4685725e-d87d-4b83-95e2-aab8c5fde82f)

The proposed change makes it look like a disabled link:
![image](https://github.com/ni/nimble/assets/1588923/795abe2e-9ffa-42dd-a4f5-90fd30fe5747)

For reasons[1] a link is emitted for this case instead of a plaintext span and it's partially styled to visually appear like plaintext. Maybe a comment update is needed to clarify the intention but I don't think the style change proposed in the PR gets closer to the goal.

[1] I believe the reason is that links can be either normal links or mentions. They are both represented in markdown as links. The way you differentiate is (partially) by the scheme. http and https are always normal links or mentions. Other schemes cannot be normal links but may be mentions, however mentions can be lazily / dynamically defined. So we can't throw away any "unsupported links" because what is supported changes dynamically. A previously unsupported URL scheme may become a supported mention scheme.

Reply:
Reverted my change to rich-text viewer link styling.

Reply:
> Maybe a comment update is needed to clarify the intention

Didn't see a response to this statement so re-opening. Did you find the existing comment clear enough (so that during refactors and updates it's not lost)?

Reply:
Sorry, I didn't read that closely enough. I realized that just reverting my change to the rich-text viewer styling isn't sufficient, because a non-href link still ends up with an underline. I tried to change the selector to affect only the non-href case (by adding `:not(:any-link)`) so I could then also remove the underline, but that selector doesn't seem to work on Chromium. Seems like this is an existing [bug](https://issues.chromium.org/issues/40623497). Instead, I modified the markdown parser to emit the `underline-hidden` attribute on a link that doesn't have an `href`. I modified the existing tests to verify this.

I have also updated the comment to be clearer about why we're styling links to look like plain text.